packages/cli/src/generator/printRunInstructions.js

```diff
@@ -22,19 +23,19 @@ function printRunInstructions(projectDir, projectName) {
     process.cwd(),
     xcodeProjectPath
   );
-  console.log(chalk.white.bold('To run your app on iOS:'));
-  console.log(`   cd ${absoluteProjectDir}`);
-  console.log('   react-native run-ios');
-  console.log('   - or -');
-  console.log(`   Open ${relativeXcodeProjectPath} in Xcode`);
-  console.log('   Hit the Run button');
+  logger.info(chalk.white.bold('To run your app on iOS:'));
+  logger.info(`   cd ${absoluteProjectDir}`);
```

Comment:
`logger.info` will prepend ` INFO ` to the message, we don't want it to be here. Can you fold it down to a single `logger.info` call with a nice template string inside?

Reply:
Fixed